selfdrive/car/subaru/carcontroller.py

```diff
@@ -20,6 +21,7 @@ def update(self, CC, CS, now_nanos):
     actuators = CC.actuators
     hud_control = CC.hudControl
     pcm_cancel_cmd = CC.cruiseControl.cancel
+    long_enabled = self.CP.openpilotLongitudinalControl
```

Comment:
I see it's already done above, but I'd avoid doing this. it just introduces an unnecessary indirection

Reply:
done